login/pam_test/test

```diff
@@ -0,0 +1 @@
+auth required pam_glome.so
```

Comment:
What is this file used for? I saw that we're also generating pam config files in lines 73 and 136.

Reply:
It was used to create a directory used for the test. Now replaced with a `custom_target` to create the required directory.